hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/api/db.json

```diff
@@ -22,7 +22,7 @@
     "datanodes": [
       {
         "hostname": "localhost2.storage.enterprise.com",
-        "uuid": "b590734e-a5f2-11ea-bb37-0242ac130002",
+        "uuid": "4712ba3d-4bb2-477a-9211-d9b50c013055",
```

Comment:
Why did we change the UUID's here?

Reply:
@ArafatKhan2198  This for Mock Data in Local to show Tooltip as it is going to decommission soon or local testing.